### PR TITLE
the model_name in model event log should be primary coming from the model.name

### PR DESCRIPTION
**Pull request by wintonzheng**
- [ ] Does this PR have impact on local development experience? If yes, make sure you have a plan and add the documentations to address issues that come with the change
- [ ] bump version
- [ ] make a release
- [ ] publish to pypi service

### CODE REVIEW

File: wyvern/components/models/model_component.py [hunk 1]
```diff
@@ -112,6 +112,7 @@ async def execute(self, input: INPUT_TYPE, **kwargs) -> MODEL_OUTPUT:
         def events_generator() -> List[ModelEvent]:
             timestamp = datetime.utcnow()
             all_events: List[ModelEvent] = []
+            model_name = model_output.model_name or self.name or self.__class__.__name__
```
Comment (suchintan) [inline wyvern/components/models/model_component.py:115]: But: it feels error prone to keep adding cases here. Should we default self.name instead?

Comment (wintonzheng) [reply wyvern/components/models/model_component.py:115]: yeah, i can deprecate model_name in model_output.

Comment (wintonzheng) [reply wyvern/components/models/model_component.py:115]: yeah, i don't like the design of `model_name under model_output`

Comment (wintonzheng) [reply wyvern/components/models/model_component.py]: @suchintan updated.

**Comment by wintonzheng** [comment]
I'm gonna merge and fix it for inventa

will do a follow up PR to deprecate model_output.model_name

**Pull request merged**